src/bp/ui-studio/src/web/reducers/selectors.ts

```diff
@@ -48,20 +48,18 @@ export const getCurrentFlowSubFlows = createSelector(
   [_getFlowsByName, _getCurrentFlow],
   (flowsByName, currentFlow: string): ParsedFlowDefinition[] => {
     return Object.keys(flowsByName)
-      .filter(name => flowsByName[name].parent === currentFlow && !isSkillFlow(name))
+      .filter(name => flowsByName[name].type === 'reusable' && !isSkillFlow(name))
       .map(name => parseFlowName(name, true))
   }
 )
 
-export const getParentFlowOutcomeUsage = createSelector(
+export const getCallerFlowsOutcomeUsage = createSelector(
   [_getFlowsByName, _getCurrentFlow],
   (flowsByName, currentFlow: string): any => {
-    const parentName = flowsByName[currentFlow]?.parent
-    if (!parentName) {
-      return []
-    }
-
-    const nodes = flowsByName[parentName].nodes.filter(x => x.flow === currentFlow)
+    let nodes = []
+    Object.values(flowsByName).forEach(flow => {
+      nodes = [...nodes, ...flow.nodes.filter(x => x.flow === currentFlow)]
+    })
```

Comment:
The selector could also be used here

Reply:
This doesn't do the same thing. It gets the nodes that call the flow instead of getting the flows that contain caller nodes.